openpype/hooks/pre_new_console_apps.py

```diff
@@ -25,7 +25,11 @@ def execute(self):
         # Set `stdout` and `stderr` to None so new created console does not
         #   have redirected output to DEVNULL in build
         self.launch_context.kwargs.update({
-            "creationflags": subprocess.CREATE_NEW_CONSOLE,
-            "stdout": None,
-            "stderr": None
+            "creationflags": subprocess.CREATE_NEW_CONSOLE
         })
+
+        if "stdout" not in self.launch_context.kwargs:
```

Comment:
Both `"stdout"` and `"stderr"` are always available, both conditins will never be met. Why you need to change this file, it should work with it's origin state?